cg/apps/balsamic/api.py

```diff
@@ -62,6 +60,7 @@ def run_analysis(self, arguments: dict, run_analysis: bool, dry: bool = False) -
         options = self.__build_command_str(
             {
                 "--account": self.account,
+                "--benchmark": "HERE_BE_DRAGONS",
```

Comment:
@Mropat here is the flag. Since you are refactoring APIs, I was not sure how much I can change.

Reply:
Also, feel free to close this PR, or merge it into other PRs as you see fitting.